### PR TITLE
Add blog post on running MongoDB workloads on pgEdge Postgres

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes FerretDB/engineering#226.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+## Setting up Postgres with pgEdge
+
+Before setting up a FerretDB instance, you need a Postgres database configured and ready.
+Then you'll pass its connection string as the `FERRETDB_POSTGRES_URI`/ `--postgres-uri flag` when you run the FerretDB instance.
+
+Start by setting up a Postgres database named `ferretdb` on pgEdge, and wait a few seconds for the database to be provisioned.
+
+![image of postgres](/img/blog/pgedge-connection.png)
+
+Once it's ready, get the connection URI credentials for the `admin` user and use it as the FERRETDB_POSTGRES_URI/ –-postgres-uri flag.
```
Comment (chilagrow) [inline website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md:40]: `FERRETDB_POSTGRES_URI`/ `--postgres-uri`, the first dash of the double dashes look weird. :thinking:

File: website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+## Connect FerretDB to Postgres using Docker
+
+Ensure you have Docker running.
+Run the FerretDB container using the following command without the connection uri from pgEdge configured to connect via the `FERRETDB_POSTGRESQL_URL`.
```
Comment (chilagrow) [inline website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md:45]: I think this sentence can be rephrased. Something like this? :thinking: 

Run the FerretDB container using the following command to connect via the `FERRETDB_POSTGRESQL_URL`. Not that `FERRETDB_POSTGRESQL_URL` is not the same as `pgEdge`'s connection string.

File: website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+Then let's run some quick `find` commands to check out our data and see the inserted book records.
+For example, let find the book "Learning SQL" using its title:
+
+```json5
+db.books.find({ title: "Learning SQL" })
+```
+
+Output:
+
+```json
```
Comment (chilagrow) [inline website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md:110]:
```suggestion
```json5
```

Applies to other places too.

File: website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+```json5
+db.books.updateOne(
+ { title: "Database Systems Concepts" },
+ { $set: { author: "Silberschatz, Korth, and Sudarshan", genre: "Technical", publication_year: 2002, isAvailable: true } },
+ { upsert: true }
+)
+```
+
+Nex, let's run a `find` command with the `sort` option to list all books sorted by the publication_year in descending order.
```
Comment (chilagrow) [inline website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md:135]:
```suggestion
Next, let's run a `find` command with the `sort` option to list all books sorted by the publication_year in descending order.
```

File: website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+ferretdb>
+```
+
+## Run MongoDB CRUD commands on FerretDB
+
+Run basic MongoDB CRUD commands on your Postgres database using FerretDB:
+
+Let's start by inserting a couple of interesting database books into a `books` collection.
+
+```json5
```
Comment (chilagrow) [inline website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md:82]:
```suggestion
```js
```

Don't we use `js` for query and `json5` for response? I think we run formatter for input such as this one and that should be marked as `js` right? Please do check.

**Review by chilagrow — COMMENTED** [review]
Great read! See my comments :hugs:

File: website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+Make sure to replace `<password>`, `<host>`, and `<port>` with your pgEdge connection details.
+
+```sh
+docker run -e FERRETDB_POSTGRESQL_URL='postgresql://admin:<password>@<host>/ferretdb?sslmode=require' -p 27017:27017 ghcr.io/ferretdb/ferretdb
+```
+
+Connect to FerretDB via `mongosh` using the following connection string:
+
+```sh
+mongosh "mongodb://admin:&lt;password>@localhost/ferretdb?authMechanism=PLAIN"
```
Comment (chilagrow) [inline website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md:55]:
```suggestion
mongosh "mongodb://admin:<password>@localhost/ferretdb?authMechanism=PLAIN"
```

File: website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md (hunk 1 above)
Comment (chilagrow) [reply website/blog/2024-11-13-run-mongodb-commands-ferretdb-pgedge-postgres-backend.md:40]:
```suggestion
Once it's ready, get the connection URI credentials for the `admin` user and use it as the `FERRETDB_POSTGRESQL_URL`/ `--postgresql-url` flag.
```

**Review by chilagrow — APPROVED** [review]

**Review by noisersup — DISMISSED** [review]

**Review by chilagrow — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**